.github/workflows/build-documentation.yml

```diff
@@ -167,6 +167,21 @@ jobs:
           # Could add pandoc and convert to github markdown
           # pandoc CFE_Users_Guide.pdf -t gfm
 
+  deploy-usersguide:
+    needs: build-usersguide
+    # Name the Job
+    name: Deploy Users Guide
+    # Set the type of machine to run on
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Cache cFS Build Environment for usersguide
+        id: cache-bundle
```

Comment:
should come up with a unique name for each cache step, maybe `cache_usersguide` and `cache_osalguide`?

Reply:
Used cache_usersguide and cache_osalguide